spec/spec_helper.rb

```diff
@@ -4,12 +4,12 @@
 include Mercury::TestUtils
 
 # the block must return a Cps
-def test_with_mercury_cps(sources, queues, **kws, &block)
+def test_with_mercury_cps(sources, queues, **kws)
   em do
     seql do
       let(:m)  { Mercury::Monadic.open(**kws) }
       and_then { delete_sources_and_queues_cps(sources, queues) }
-      and_then { block.call(m) }
+      and_then { yield(m) }
```

Comment:
This is one I struggle with. Sometimes when a block is required (or I want to give it a name that indicates what it does), I want to see it in the method signature.

So this one isn't as straightforward for me.